src/mongocrypt.h.in

```diff
@@ -408,6 +408,22 @@ bool
 mongocrypt_setopt_schema_map (mongocrypt_t *crypt,
                               mongocrypt_binary_t *schema_map);
 
+/**
+ * Set a local EncryptedFieldConfigMap for encryption.
+ *
+ * @param[in] crypt The @ref mongocrypt_t object.
+ * @param[in] schema_map A BSON document representing the EncryptedFieldConfigMap supplied by
+ * the user. The keys are collection namespaces and values are EncryptedFieldConfigMap documents. The
+ * viewed data copied. It is valid to destroy @p efc_map with @ref
+ * mongocrypt_binary_destroy immediately after.
+ * @pre @p crypt has not been initialized.
```

Comment:
Maybe this is normal markup, but I can't quite parse "@pre @p".

Reply:
Weird as they are `@pre` is [Doxygen precondition](https://www.doxygen.nl/manual/commands.html#cmdpre) and `@p` is [Doxygen typewriter font](https://www.doxygen.nl/manual/commands.html#cmdp).